pytest/tests/sanity/rpc_hash.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+"""Test computing block hash from data provided by JSON RPCs."""
+
+import hashlib
+import json
+import pathlib
+import sys
+import typing
+import unittest
+
+import base58
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+
+from configured_logger import logger
+import cluster
+import utils
+import serializer
+
+import messages
+import messages.crypto
+import messages.block
+
+
+def serialize(msg: typing.Any) -> bytes:
+    return serializer.BinarySerializer(messages.schema).serialize(msg)
+
+
+def compute_block_hash(header: typing.Dict[str, typing.Any],
```

Comment:
if we have this function already - - how about we extend this test a little?

we could create couple 'static' json files - load them and check that the hashes computed by this are correct.
(and those json files could come from 'real' blocks on the blockchain)

(yes, this means that we'd have a 'test' for a 'test' method ;-) - but long term, we might want to move this function into some python utils.

 (and there are a bunch of TODOs below - which we'll be able to cover with these 'static' tests)

Reply:
do you plan to also use/test the code from: https://github.com/near/nearcore/blob/master/pytest/lib/lightclient.py#L33 ?

Reply:
Regarding light client, I am planning to change it so that it uses `messages.schema` rather than rolling the schema on its own, but otherwise I don’t have any plans to use the code here.

As for TODOs and taking header from mainnet, blocks on mainnet don’t have those fields set either.  Challenges aren’t implemented at